executor/runtime/types/types.go

```diff
@@ -319,6 +320,9 @@ func (n *NetworkConfigurationDetails) ToMap() map[string]string {
 	if n.EniIPv6Address != "" {
 		m["EniIPv6Address"] = n.EniIPv6Address
 	}
+	if n.ElasticIPAddress != "" {
+		m["ElasticIPAddress"] = n.ElasticIPAddress
```

Comment:
Huh, do you know if we are ever going to have ElasticIPv6Addresses?

Reply:
Not sure - they don't exist today: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html